Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,150 @@ label mas_show_unseen:
     m 1esa "Just give me a second.{w=0.5}.{w=0.5}.{nw}"
     m 3hua "There you go!"
     return
+
+#Whether or not the player can code in python
+default persistent._mas_pm_can_code_python = None
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_installed_python",
+            category=['misc'],
+            prompt="I installed Python",
+            pool=True,
+            unlocked=True
+        )
+    )
+
+label monika_installed_python_code_level_menu:
+    m 1etc "Have I been underestimating your coding skills?{nw}"
+    $ history_list.pop()
+    menu:
+        m "Have I been underestimating your coding skills?{fast}"
+
+        "Yes":
+            m 1hksdlb "Ahaha, I'm sorry, [player]!"
+            m 3rka "I didn't mean to- I just never thought to ask."
+            $ persistent._mas_pm_can_code_python = True
+            return True
+
+        "No":
+            m 1ekb "Okay, good."
+            m 3eka "I just wanted to make sure I wasn't assuming your skill level."
+            $ persistent._mas_pm_can_code_python = False
+            return False
+
+label monika_installed_python_version_menu:
+    m 1eub "What version of Python did you download?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "What version of Python did you download?{fast}"
+
+        "Python 2":
+            m 1hub "Great!"
+            m 1eub "That's the version of Python I use."
+            return True
+
+        "Python 3":
+            m 1hksdlb "..."
+            m 1rksdla "[player], I use the other version of Python..."
+            return False
+
+        #NOTE: this could probably use a both path (you can change these returns to be 1, 2, and 3 to check)
+
+label monika_installed_python:
+    m 1sub "You did?"
+    m 1hub "That's great, [player]!"
+    m 1eub "What did you download it for?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "What did you download it for?{fast}"
+
+        "A personal project.":
+            m 1hub "How fun!"
+            m 3hub "Good luck with whatever you're programming!"
+            m 1eub "I'll teach you as much as I can!"
+            m 3ekbfa "Once you learn the basics, could you help me out with a few things?"
+            m 1hub "I know you'll be even better than me one day, and you could even teach me more about programming at that point!"
+            m 1eub "Ehehe!"
+            m 3hua "Just ask me if you ever need help with anything!"
+
+            call monika_installed_python_version_menu
+            $ installed_py2 = _return
+
+            if installed_py2:
+                m 1hua "You'll do amazing things, I just know it!"
+
+            else:
+                m 3rksdlb "So we can't really help each other that much..."
+                m 1hub "But that's okay!"
+                m 1eub "I hope you have fun with your project."
+
+        "To help you come closer to our reality.":
```

Comment:
Could be fitting for Monika to reference other 2 topics: the one where she asks if you've contributed (monika_contribute) and the one where she asks about rpy files (monika_rpy_files).